src/Badges.sol

```diff
@@ -82,6 +83,12 @@ contract Badges is
     transferOwnership(_nextOwner);
   }
 
+  function refreshMetadata(string[] memory _specUris) external onlyOwner {
+    for (uint256 i = 0; i < _specUris.length; i++) {
+      emit RefreshMetadata(_specUris[i]);
```

Comment:
Can we not emit a single event with Array? would be more gas efficient, no? 
need to also validate if arrays can be handled in subgraph